rc/windowing/zellij.kak

```diff
@@ -0,0 +1,69 @@
+# https://zellij.dev/
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+provide-module zellij %{
+
+# ensure we're running under zellij
+evaluate-commands %sh{
+    [ -z "${client_opt_windowing_modules}" ] || [ -n "$ZELLIJ" ] || echo 'fail zellij not detected'
```

Comment:
`[ -z "${client_opt_windowing_modules}" ]` always returns 0. It should be `client_env..` right? Also, if we are checking the client, not server, we probably should check it on any command invocation. Because module will be loaded only once. So having this in a separate, callable command would probably make more sense.

Reply:
Sorry, I might copy the logic from an old `tmux.kak`, I will change it to `"${kak_opt_windowing_modules}"` as the latest one, does it OK?